platform/darwin/src/MGLMapCamera.h

```diff
@@ -63,10 +78,47 @@ MGL_EXPORT
     The value `180` means the top of the map points due south, and so on.
  */
 + (instancetype)cameraLookingAtCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate
-                                   fromDistance:(CLLocationDistance)distance
+                          correctlyFromDistance:(CLLocationDistance)distance
```

Comment:
Would it be possible avoid the `correctly` word? what are your thoughts on `fromLineOfSightDistance`?

Reply:
I considered `fromLineOfSightDistance`, but it bridges to Swift as `fromLineOfSight`, which is unfortunate.

Reply:
Another possibility would be `acrossDistance`, which bridges to Swift elegantly as `across`.

Reply:
I'm inclined for `acrossDistance` instead.

Reply:
I like having the word distance in there - can we use NS_SWIFT_NAME in this case? To throw some more out there, perhaps `withSubjectDistance`, `fromViewDistance`, `withCameraAtDistance`.

Reply:
> I like having the word distance in there

Forcing the word “distance” would violate [Swift API naming guidelines](https://swift.org/documentation/api-design-guidelines/#omit-needless-words). The Swift compiler is doing the right thing here, since the type is `CLLocationDistance`. I’ll go with `acrossDistance` in Objective-C, which bridges to Swift as `across`.